### PR TITLE
Lavaland Ruins

### PR DESCRIPTION
**Pull request by MMMiracles**
pod
_A capsule shelter, once sanctuary for the common miner, has become their own tomb._

memes
_A long-since abandoned outpost nestled within the region of the hellish planet. These don't look like syndicate uniforms.._

memes2
_An ancient tomb housing the remains of whoever was once buried here. You swear you hear some rattling coming from inside the structure._

Also includes a big o' chasm and 2 variants of a ravine (crevice?), one horizontal while the other is vertical. Lavaland needs more big scary obstacles that isn't lava or unbreakable walls.

i did it.

### CODE REVIEW

**Comment by Armhulen** [comment]
I like it.

**Comment by ChangelingRain** [comment]
Wrong door type, that's a vertical door you don't want vertical there

**Comment by MMMiracles** [comment]
is anyone really going to notice that the door goes in a different direction compared to other pods because i just closed dream maker and it takes like 2 minutes to fully load in uggh.

**Comment by KorPhaeron** [comment]
Ruin list really needs a way to weight the picks by size, this tiny box counts the same as the entire wishgranter maze as far as the game is concerned

**Comment by Cheridan** [comment]
imo i wouldn't put the goliath there since lavaland's got plenty already. and might be weird going "oh an old survival pod i know a goliath is right over here"

**Comment by MMMiracles** [comment]
@Cheridan i mean the footprints and blood probably give it away already, seeing how it'd be such a specific look. goliath is also dead to show the whole 'miner died after fighting off big scary monster'

**Comment by KorPhaeron** [comment]
Only think this is a good idea if @coiax finishes the ruin cost PR, otherwise it's a waste of a slot

**Comment by MMMiracles** [comment]
i got bored waiting for the cost PR so i added another ruin that is 100% PC and can't be denied due to hurting someone's feelings, trigger warnings included

**Comment by PKPenguin321** [comment]
Should have saved it for a separate PR
There is a lot of loot in that ruin

**Comment by MMMiracles** [comment]
surrounded by a 3-thick lava moat and the best thing you can get out of is a moist nugget and a few shitty pistols.

also with the current theme of loot from lavaland it doesn't seem very overkill, adding in that i'll be giving it a modest price for when the ruin pricing PR goes through.

**Comment by PKPenguin321** [comment]
Ok but blueprints tho? And what look like hulk injectors?

**Comment by MMMiracles** [comment]
pamphlet with blueprint icon 

1 temp hulk injector, not a permanent one 

1 permanent dwarfism injector

**Comment by MMMiracles** [comment]
paging our lord and savior design lead @KorPhaeron to make sure i'm staying within the deep lore that is lavaland with these ruins.

**Comment by KorPhaeron** [comment]
Well we already have gym full of wrestling zombies so a secret nazi base in hell probably fits

**Comment by coiax** [comment]
@MMMiracles please observe the quality lore and names given to ruins in https://github.com/tgstation/-tg-station/pull/17366

**Comment by KorPhaeron** [comment]
@MMMiracles 

Conflicts

**Comment by MMMiracles** [comment]
@KorPhaeron i did it

**Comment by coiax** [comment]
I have just read the descriptions and the content and I enjoyed it.

Just a fair warning, with 3x zero cost chasms, they will almost guarantee an appearance on lavaland each round. However, I think this is almost certainly fun and entertaining, and only has a very small chance of reacting with some sort of endless ruin placement turning lavaland into a bottomless pit. 10/10

**Comment by MMMiracles** [comment]
i heard merging my PRs makes you cool.

**Comment by KorPhaeron** [comment]
There is literally no way to cross chasms right (other than having another miner throw you) now would be the biggest issue with plopping down so many of them

**Comment by MMMiracles** [comment]
the chasm has a pathway across the mid-section while the ravines are splits that can be walked around, unless ruin placement decides lavaland needs 100% more holes.

**Comment by KorPhaeron** [comment]
Monsters also dont know to avoid holes, and its very easy to run into them as lizards/golems since your light lags behind you.  I will test merge later

**Comment by ChangelingRain** [comment]
I don't like the huge base also having a lava river around it, it'd work better just being in a cavern(and be easier to go around)

**Comment by MMMiracles** [comment]
Lake is meant to be an obstacle to surpass if you want the guns inside. I left plenty of open area around it so it's not like it's hard to just go around.

**Comment by MMMiracles** [comment]
@KorPhaeron the fact there hasn't been any kind of implementation yet for AIs to path around dangerous tiles is rather silly

**Comment by MMMiracles** [comment]
i don't feel like having the potential of holeland holding up the PR so now they can only spawn once, same with the outpost.

i think i have an idea for another PR to allow crossing a chasm that doesn't involve throwing people over but for now meh.

**Comment by MMMiracles** [comment]
m e m e s

**Comment by MMMiracles** [comment]
@KorPhaeron 
@duncathan 
@RemieRichards 
@otherpeoplewiththeabilitytomergeprs

**Comment by KorPhaeron** [comment]
This PR is stuck in the limbo of me not really being excited about it but not activelyet disliking it to the point I would close it.

I will merge soon probably sorry

**Comment by MMMiracles** [comment]
you're denying the playerbase space nazis

literally anti-hitler

**Comment by KorPhaeron** [comment]
I like space nazis but its mostly just a couple of R-wall boxes with empty rooms your away mission was way cooler

**Comment by MMMiracles** [comment]
nazi hq was cooler because it was an away mission, which meant i could throw balance out the window and replace it with more nazis

theres no room for giant nazi base with mecha-hitler at the moment, sadly enough

**Comment by KorPhaeron** [comment]
>theres no room for giant nazi base with mecha-hitler at the moment, sadly enough

Not with that attitude. Remember that the wishgranter maze fits in lavaland quite easily.

**Comment by MMMiracles** [comment]
maybe.

just maybe.

**Comment by MMMiracles** [comment]
@KorPhaeron psst ill make you a deal

if you merge this pr ill take actual time and effort into restoring the hq and making it a ruin

do it for the community, they need the nazis

**Comment by PKPenguin321** [comment]
Do it kor its a good deal

**Comment by MMMiracles** [comment]
forreal though i kinda need this PR to go through if i want to actually start looking into remaking the HQ since it has the clothes and mobs already.

**Comment by KorPhaeron** [comment]
fiiiiiiiiiiiiine

File: code/modules/mob/living/simple_animal/hostile/reenactors.dm [hunk 1]
```diff
@@ -0,0 +1,103 @@
+/*
+-!WARNING!-
+-POTENTIAL TRIGGER-INDUCING ITEMS AHEAD-
+-TREAD WITH CAUTION-
+*/
+
+/mob/living/simple_animal/hostile/wwii
+	name = "World War II Reenactor"
```
Comment (KorPhaeron) [inline code/modules/mob/living/simple_animal/hostile/reenactors.dm]: hell/moon nazis are so much cooler than reenactors though

**Comment by KorPhaeron** [comment]
Would it be a lot of trouble to put HOLELAND in another PR so I can test merge that because I forgot to ever do it, the rest of this isfine but I'm still worried about lmao3chasm happening

File: code/modules/mob/living/simple_animal/hostile/reenactors.dm (hunk 1 above)
Comment (MMMiracles) [inline code/modules/mob/living/simple_animal/hostile/reenactors.dm]: cheridan himself said in the original away mission PR he didn't like the idea of adding nazis so this was the compromise.

convince him ss13 needs real nazis and i'll change them back

**Comment by KorPhaeron** [comment]
I'd testmerge now but I think we're doing a maratho nfollowed by 2 days of the server being down

**Comment by MMMiracles** [comment]
@KorPhaeron all the chasms have a way around them built into the ruin itself (ravines can be walked around and the chasm has a natural bridge across it.

 I also changed it at most only spawn each once, which i doubt is gonna cause much issue in terms of blockage past what the lava already does

**Comment by KorPhaeron** [comment]
yeah but you can cross lava but I'm a lazy piece of shit and its literally impossible to interact with chasms, even with an RCD

**Comment by MMMiracles** [comment]
fine fine ill get rid of holeland

**Comment by MMMiracles** [comment]
i sure hope this goes through because i already fucked up my local with trying to fix HQ and i don't feel like rebasing

**Comment by MMMiracles** [comment]
@KorPhaeron it PASSED

NA-ZI

**Comment by MMMiracles** [comment]
memes

**Comment by PKPenguin321** [comment]
:clap: :tada: :balloon: :birthday: :clap:

**Comment by Cheridan** [comment]
what the fuck joan

**Pull request merged**